public/app/features/dashboard/dashgrid/DashboardLibrary/DashboardLibrarySection.tsx

```diff
@@ -84,117 +96,98 @@ export const DashboardLibrarySection = () => {
     locationService.push(templateUrl);
   };
 
-  if (!templateDashboards?.length) {
-    return null;
-  }
-
   return (
-    <Box borderRadius="lg" borderColor="strong" borderStyle="dashed" padding={4} flex={1}>
-      <Stack direction="column" alignItems="center" gap={2}>
-        <Text element="h3" textAlignment="center" weight="medium">
-          <Trans i18nKey="dashboard.empty.start-with-suggested-dashboards">
-            Start with a pre-made dashboard from your data source
+    <Stack direction="column" gap={2}>
```

Comment:
```suggestion
<Stack direction="column" gap={2} height="100%">
      <Box flex={1}>
```
this makes the pagination buttons to be placed at the bottom